Android/app/src/main/java/fashiome/android/activities/EntryActivity.java

```diff
@@ -0,0 +1,21 @@
+package fashiome.android.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+/**
+ * Created by dsaha on 3/18/16.
+ */
+public class EntryActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        Intent launchIntent = new Intent(EntryActivity.this, IntroAndLoginActivity.class);
```

Comment:
Why do you need an EntryActivity just to start an intent to IntroAndLoginActivity ? Cant you directly make the IntroAndLoginActivity as the LAUNCHER activity